packages/peregrine/lib/talons/Header/__tests__/useStoreSwitcher.spec.js

```diff
@@ -233,31 +255,125 @@ describe('handleSwitchStore updates url with configured store code and url suffi
             assign: jest.fn()
         };
 
-        handleSwitchStore('store1');
+        handleSwitchStore('store2');
 
+        // .html => null
```

Comment:
Is this comment wrong or the assertion innaccurate?

Reply:
Good catch, comment is wrong. This test is to check for store code, not the suffix